Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -115,6 +127,21 @@ public static async Task<T> GoToPageAsync<T>(this UITestContext context, string
         return page;
     }
 
+    public static async Task<T> GoToAdminPageAsync<T>(this UITestContext context, string relativeUrl = null)
+        where T : PageObject<T>
+    {
+        var page = context.ExecuteLogged(
+            $"{typeof(T).FullName} - {context.AdminUrlPrefix + relativeUrl}",
+            typeof(T).FullName,
+            () => context.Scope.AtataContext.Go.To<T>(url: context.GetAbsoluteUri(context.AdminUrlPrefix + relativeUrl).ToString()));
```

Comment:
I don't get this. Are you suggesting to replace `GetAbsoluteUri()` with `GoToAdminRelativeUrlAsnyc()` ? 
Because that would save the context.AdminUrlPrefix parameter but the first is a `Uri` returning method and the second is a `Task` so I can't really replace one with the other.

Reply:
No. You have `context.GetAbsoluteUri(context.AdminUrlPrefix + relativeUrl)` copied in two places. This logic will get more involved with the `UriBuilder` usage I mentioned elsewhere. So, have it only in a single place and reuse it.

Reply:
I did remove this line and replaced it with UriBuilder could you check?